nevergrad/functions/functionlib.py

```diff
@@ -63,6 +65,43 @@ def _short_repr(self) -> str:
         return "Photonics"
 
 
+class ML_tuning(ExperimentFunction):
+    """Class for generating ML hyperparameter tuning problems.
+    """
+    def __init__(self, problem_type: str):
+        self.name = problem_type
+        self._parameters = {x: y for x, y in locals().items() if x not in ["__class__", "self"]}
+        self.rng = np.random.RandomState(1)
```

Comment:
I would advise strongly against having a random generator here unless there is a clear reason you do not want to instantiate several different versions.
You could use the parametrization random generator instead (parameter.random_state)

Reply:
👍 thx :-)